src/views/DataExploration/components/PageContent/tabs/Summary/DemographicGraphCard/index.tsx

```diff
@@ -93,7 +92,7 @@ const DemographicsGraphCard = ({ id, className = '' }: OwnProps) => {
               <PieChart
                 title={intl.get('screen.dataExploration.tabs.summary.demographic.sexTitle')}
                 data={sexData}
-                onClick={(datum) => addToQuery('sex', datum.id as string, history)}
+                onClick={(datum) => addToQuery('sex', datum.id as string)}
```

Comment:
do the fields `sex`, `race` and `ethnicity` exist in an enum? If not, do you think we should create one?

Reply:
Yes I think we should definitely avoid passing a hardcoded string as a key. Not sure if that follows what you had in mind, but I created a SUMMARY_KEY enum to handle the possible keys we use in the Summary section.